tests/testthat/test-buffer.R

```diff
@@ -0,0 +1,179 @@
+chosen_points <- sf::st_as_sf(
+  data.frame(x = c(0, 1, 3), y = 1),
+  coords = c("x", "y"),
+  # Any projected CRS should be fine here
+  crs = 2249
+)
+
+test_that("buffering selects the expected points", {
+
+  skip_if_offline()
+  sf::sf_proj_network(enable = TRUE)
+
+  skip_if_not(sf::sf_use_s2())
+
+  # These points fall along a number line: point 1 is 1 away from point 2,
+  # point 3 is 2 away from point 2
+
+  # Using a projected CRS (so no geographic weirdness), that means buffering
+  # should be conceptually straightforward: points X units away should be
+  # "caught" by any radius or buffer (or the two combined) >= X
+
+  # No buffer or radius: only the selected point (2) should be in test:
+  expect_identical(
+    buffer_indices(chosen_points, list(2), 0, 0),
```

Comment:
could you name the arguments to buffer_indices(), please? that makes, e.g. "no buffer or radius" more explicit